sssd_test_framework/utils/authentication.py

```diff
@@ -397,44 +407,82 @@ def passkey(self, username: str, *, pin: str | int, device: str, ioctl: str, scr
             mode="a=rx",
         )
 
+        #import pdb; pdb.set_trace()
         result = self.host.ssh.expect(
             rf"""
+            # Disable debug output
+            # exp_internal 0
+
+            proc exitmsg {{ msg code }} {{
+                # Close spawned program, if we are in the prompt
+                catch close
+
+                # Wait for the exit code
+                lassign [wait] pid spawnid os_error_flag rc
+
+                puts ""
+                puts "expect result: $msg"
+                puts "expect exit code: $code"
+                puts "expect spawn exit code: $rc"
+                exit $code
+            }}
+
             # It takes some time to get authentication failure
             set timeout {DEFAULT_AUTHENTICATION_TIMEOUT}
             set prompt "\n.*\[#\$>\] $"
+            set command "{command}"
 
-            spawn "{playback_umockdev}"
+            spawn {playback_umockdev}
```

Comment:
You can probably add the quotes back here.   I removed them when troubleshooting an issue but, forgot to put them back.

Reply:
updated